containers/src/Kokkos_ScatterView.hpp

```diff
@@ -695,6 +695,11 @@ class ScatterView<DataType, Layout, ExecSpace, Op, ScatterNonDuplicated,
   template <typename DT, typename... RP>
   void contribute_into(View<DT, RP...> const& dest) const {
     typedef View<DT, RP...> dest_type;
+    // We need to wait in case the scatterview was used in a different
```

Comment:
Should `contribute_into` have an overload that takes an execution space instance?